src/shared/layout/EventLayout.js

```diff
@@ -10,23 +10,28 @@ export const EventLayout = ({
   icon,
   button1,
   button2,
+  isMobileView,
 }) => {
   let content;
 
   if (events.length === 0) {
     content = (
       <div>
-        <p>{`Stay tuned :)`}</p>
+        <p className={isMobileView && "text-center"}>{`Stay tuned :)`}</p>
       </div>
     );
   } else {
     content = events.map((event) => (
       <div className="grid gap-y-8">
+        {!isMobileView && (
+          <div>
+            <p>{event.desc}</p>
+          </div>
+        )}
         <div>
-          <p>{event.desc}</p>
-        </div>
-        <div>
-          <div className="event-img">{event.img}</div>
+          <div className="event-img max-w-[80%] md:max-w-[50%] m-auto lg:max-w-[95%] lg:m-0">
```

Comment:
Can you replace `max-w-` with `w-` ? If you want to set a fixed width just use `w-` instead of `max-w-` because image can be smaller than the `max-w-`

Reply:
Good point